inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -606,69 +605,67 @@ struct jit_uni_mvn_kernel_f32 : public jit_uni_mvn_kernel, public jit_generator
 //////////////////////////////////////////////////////////////////////////////////
 
 MKLDNNMVNNode::MKLDNNMVNNode(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache)
-        : MKLDNNNode(op, eng, cache), epsMode_(insideSqrt) {}
-
-void MKLDNNMVNNode::getSupportedDescriptors() {
-    if (!descs.empty())
-        return;
-
+        : MKLDNNNode(op, eng, cache) {
     std::string errPrefix = "MVN node with name '" + getName() + "' ";
 
-    auto cnnLayer = getCnnLayer();
-    if (cnnLayer == nullptr)
-        THROW_IE_EXCEPTION << errPrefix << "does not have CNN layer.";
-
-    if (getParentEdges().size() > 2)
-        THROW_IE_EXCEPTION << errPrefix << "has incorrect number of input edges.";
-
-    if (getChildEdges().empty())
+    if (op->get_output_size() == 0)
         THROW_IE_EXCEPTION << errPrefix << "has incorrect number of output edges.";
 
-    const auto& numOfDims = getParentEdgeAt(0)->getDims().ndims();
-    if (numOfDims < 1 || numOfDims > 5)
-        THROW_IE_EXCEPTION << errPrefix << "doesn't support input with size of dimensions: " << numOfDims;
-
-    across_channels = false;
-    if (getParentEdges().size() == 1) {
-        across_channels = cnnLayer->GetParamAsBool("across_channels");
+    const auto& inDataShapeSize = op->input_value(0).get_shape().size();
+    if (inDataShapeSize < 1 || inDataShapeSize > 5)
+        THROW_IE_EXCEPTION << errPrefix << "doesn't support input with size of dimensions: " << inDataShapeSize;
```

Comment:
We should throw THROW_IE_EXCEPTION_WITH_STATUS(NOT_IMPLEMENTED) in case parameters are valid but not supported by the implementation. This will allow common logic to fallback on ngraph reference implementation.

Reply:
Fixed